WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/FeeLineDetailsViewModelTests.swift

```diff
@@ -209,4 +206,34 @@ final class FeeLineDetailsViewModelTests: XCTestCase {
         viewModel.saveData()
         XCTAssertEqual(savedFeeLine?.total, "20.00")
     }
+
+    func test_view_model_amount_placeholder_is_not_empty_string() {
```

Comment:
nit: You could use a name like `test_view_model_amount_placeholder_has_expected_value` since you're specifically checking that it is set to `"0"` (and it could, in theory, fail because the value is set to a different number rather than an empty string).